dev-python/awkward/awkward-2.8.1.ebuild

```
@@ -0,0 +1,57 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=hatchling
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Manipulate JSON-like data with NumPy-like idioms"
+HOMEPAGE="
+	https://github.com/scikit-hep/awkward
+	https://pypi.org/project/awkward/
+	https://doi.org/10.5281/zenodo.4341376
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	~dev-python/awkward-cpp-45[${PYTHON_USEDEP}]
+	$(python_gen_cond_dep '
+		>=dev-python/importlib-metadata-4.13.0[${PYTHON_USEDEP}]
+	' python3_11)
```

```suggestion
	' 3.11)
```

Always use stdlib versions, unless you really don't mean it to apply to PyPy (even if PyPy is not explicitly supported right now).

---

This is nice, thanks. I did not know that works too (now?).